exchanges/account/account.go

```diff
@@ -167,26 +169,30 @@ func GetBalance(exch, subAccount string, creds *Credentials, ai asset.Item, c cu
 		return nil, fmt.Errorf("%s %w", exch, ErrExchangeHoldingsNotFound)
 	}
 
-	subAccounts, ok := accounts.SubAccounts[*creds]
+	subAccounts, ok := accounts.subAccounts[*creds]
 	if !ok {
 		return nil, fmt.Errorf("%s %s %w",
 			exch, creds, errNoCredentialBalances)
 	}
 
-	bal, ok := subAccounts[key.SubAccountCurrencyAsset{
+	assets, ok := subAccounts[key.SubAccountAsset{
 		SubAccount: subAccount,
-		Currency:   c.Item,
 		Asset:      ai,
 	}]
 	if !ok {
 		return nil, fmt.Errorf("%s %s %s %s %w",
 			exch, subAccount, ai, c, errNoExchangeSubAccountBalances)
 	}
+	bal, ok := assets.balances[c.Item]
+	if !ok {
+		return nil, fmt.Errorf("%s %s %s %s %w",
+			exch, subAccount, ai, c, errNoExchangeSubAccountBalances)
+	}
 	return bal, nil
 }
 
-// Update updates holdings with new account info
-func (s *Service) Update(incoming *Holdings, creds *Credentials) error {
+// Load loads the holdings with new account info
+func (s *Service) Load(incoming *Holdings, creds *Credentials) error {
```

Comment:
🥃 I don't think this naming improves things.
Maybe you could share your thinking about the distinction?

I think in my head, `Save` is when you give it everything to store, `Load` is when you want it to go get something, and `Update` is when you give it a partial update.

So in this case, I'm thinking you renamed `Update` to `Load` because it's a full dataset, and not a partial?

If so, need to make that clearer in the comment, because not everyone will assume `Load` means clobber.

I'd vote for `Save` 😄 

Can ignore.

Reply:
For me, both "Save" and "Load" work. If there are more votes for one, I'll update it accordingly.

Reply:
I'm a fan of the original `Update` since we use that terminology with other packages though I'm curious as to your thoughts on preferring Load @junnplus

Reply:
@thrasher- Considering that asset updates will be incrementally handled via WebSocket in the future, the name `Update` better aligns with the incremental update approach.

Reply:
@gbjk Okay, I just felt that Update wasn't the best fit here. I'll change it to Save.